website_sale_product_minimal_price/models/product_template.py

```diff
@@ -1,120 +1,81 @@
 # Copyright 2019 Tecnativa - Sergio Teruel
+# Copyright 2020 Tecnativa - Carlos Roca
+# Copyright 2020 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-from odoo import fields, models, tools
+from odoo import models
+import math
 
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
+    def _get_cheapest_info(self, pricelist):
+        """Helper method for getting the variant with lowest price."""
+        # TODO: Cache this method for getting better performance
+        self.ensure_one()
+        context = dict(self.env.context, pricelist=pricelist.id)
+        min_price = math.inf
+        product_id = False
+        add_qty = 0
+        has_distinct_price = False
+        for product in self.product_variant_ids:
+            for qty in [1, math.inf]:
```

Comment:
@CarlosRoca13 I think that the float_compare method crash with inf... @pedrobaeza 2 days ago changed inf to 99999...